patches/server/0288-BlockDestroyEvent.patch

```diff
@@ -22,21 +22,25 @@ index 178509c545f2872174af501bdcec3314f703739c..d315a31f6b489ee88f81eb3d1b358c92
  import net.minecraft.server.MinecraftServer;
  import net.minecraft.server.level.FullChunkStatus;
  import net.minecraft.server.level.ServerLevel;
-@@ -669,8 +670,21 @@ public abstract class Level implements LevelAccessor, AutoCloseable {
+@@ -669,8 +670,25 @@ public abstract class Level implements LevelAccessor, AutoCloseable {
              return false;
          } else {
              FluidState fluid = this.getFluidState(pos);
 +            // Paper start - while the above setAir method is named same and looks very similar
 +            // they are NOT used with same intent and the above should not fire this event. The above method is more of a BlockSetToAirEvent,
 +            // it doesn't imply destruction of a block that plays a sound effect / drops an item.
 +            boolean playEffect = true;
++            BlockState effectType = iblockdata;
```

Comment:
I don't see this actually being used for the effect. You need to change the BlockState that goes into the levelEvent call